packages/core/src/types/config/access-control.ts

```diff
@@ -133,25 +128,44 @@ export type ListAccessControl<ListTypeInfo extends BaseListTypeInfo> =
 export type IndividualFieldAccessControl<Args> = (args: Args) => MaybePromise<boolean>;
 
 export type FieldCreateItemAccessArgs<ListTypeInfo extends BaseListTypeInfo> =
-  CreateItemAccessArgs<ListTypeInfo> & { fieldKey: string };
+  BaseAccessArgs<ListTypeInfo> & {
+    operation: 'create';
+    fieldKey: string;
+    /**
+    * The input passed in from the GraphQL API
+    */
+    inputData: ListTypeInfo['item'];
+  };
 
 export type FieldReadItemAccessArgs<ListTypeInfo extends BaseListTypeInfo> =
   BaseAccessArgs<ListTypeInfo> & {
     operation: 'read';
     fieldKey: string;
+    /**
+    * The item being read
+    */
     item: ListTypeInfo['item'];
   };
 
 export type FieldUpdateItemAccessArgs<ListTypeInfo extends BaseListTypeInfo> =
-  UpdateItemAccessArgs<ListTypeInfo> & { fieldKey: string };
+  BaseAccessArgs<ListTypeInfo> & {
+    operation: 'update';
+    fieldKey: string;
+    /**
+    * The item being updated
+    */
+    item: ListTypeInfo['item'];
+    /**
+    * The input passed in from the GraphQL API
+    */
+    inputData: ListTypeInfo['inputs']['update'];
+  };
```

Comment:
I opted for explicit types here, they are only related to the list access control functions by convention